fsharp-backend/src/ExecHost/ExecHost.fs

```diff
@@ -207,16 +207,6 @@ let initSerializers () =
   Json.Vanilla.allow<CTPusher.Payload.New404> "Pusher"
   Json.Vanilla.allow<CTPusher.Payload.AddOpV1> "Pusher"
   //Json.Vanilla.allow<CTPusher.Payload.AddOpV1PayloadTooBig> "Pusher" // this is so-far unused
-  Json.Vanilla.registerConverter (ClientTypes.Converters.STJ.WorkerStateConverter())
-  Json.Vanilla.allow<CTPusher.Payload.UpdateWorkerStates> "Pusher"
-
-  // allow serialization of types used in Pusher.com payloads
```

Comment:
(apparently I had created duplicate `.allow` statements here for pusher.com payloads)